.vscode/tasks.json

```diff
@@ -0,0 +1,83 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Develop Frontend",
+      "type": "gulp",
+      "task": "develop-app",
+      "problemMatcher": {
+        "base": "$tsc",
+        "background": {
+          "activeOnStart": true,
+          "beginsPattern": "Using gulpfile",
```

Comment:
This is not really the "beginsPattern" of a background compilation task. But it will work like this.

I looked into being able to print a "Changes detected, compiling" log line, but was unable to find out how from [the Node api docs](https://webpack.js.org/api/node/#watching).